### DISCUSSION TITLE
C# backend ,error in the project

### DISCUSSION THREAD

**Discussion by NatashkaJK** [community/community#127798] [Programming Help]
### Body

Hello everyone, I was working on the backend of a website in C# and when I updated the project build versions, this error appeared. How to solve it?
I have 2 days left until the project is due, I'm desperate
Severity    Code    Description    Project    File    Line    Suppression State    Details
Error (active)    CS1980    Cannot define a class or member that utilizes 'dynamic' because the compiler required type 'System.Runtime.CompilerServices.DynamicAttribute' cannot be found. Are you missing a reference?    1_Views_Account_Login.cshtml    C:\Users\LimBo\source\repos\SneackersShop\Views\Account\Login.cshtml    28

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by davevad93** [comment]
Hi @NatashkaJK ,

That error is really annoying and may occur when updating a project's build version, that's why I usually never update the build or dependencies while working on a project because those kind of errors may happen. I guess you're using `Visual Studio 2022` or `2019`, so try with the following steps, when completing a step check first whether it's fixed, if not then try with the next step.

1. Exclude and re-include the file that returns the error in your project, which seems to be a `cshtml` file.
2. Close your project's solution and rebuilt it.
3. If the above solution didn't work repeat the same step re-targeting the framework back to the version that worked fine.
4. Open your `csproj` file and check if there's some issue with your build configuration, some dependencies might point to old libraries.

By taking one of the first two steps you should fix it.
